daemon/sailfish-secretsd.privileges

```diff
@@ -0,0 +1,2 @@
+# Tells invoker to run sailfish-secretsd as nemo:privileged
```

Comment:
What is this file for?
I didn't find it in .spec file. Don't we put it on file system? Is this file included in package?